cmd/snap/cmd_auto_import.go

```diff
@@ -125,9 +124,16 @@ func doUmount(mp string) error {
 
 type cmdAutoImport struct{}
 
-var shortAutoImportHelp = i18n.G("Imports assertions from mounted devices")
+var shortAutoImportHelp = i18n.G("Searches devices for actionable information")
+
+var longAutoImportHelp = i18n.G(`
+The auto-import command searches available devices and mounts them looking
```

Comment:
s/available devices/specified block devices/ 

From my earlier reading we actually have to pass the devices to mount explicitly. There's no enumeration going on here.